boost.sh

```diff
@@ -948,24 +966,34 @@ buildBoost_Linux()
     mkdir -p $OUTPUT_DIR
     echo > ${OUTPUT_DIR}/linux-build.log
 
-    for BITS in 64 32; do
+    for BITS in 32 64; do
```

Comment:
Why did you reverse the order here?

Reply:
no specific reason...

Reply:
Lets keep 64 bits a priority then. We want them built first so we can start consuming/trying them right away.